core/src/blocktree.rs

```diff
@@ -1215,7 +1222,7 @@ impl Blocktree {
             self.put_coding_blob_bytes_raw(slot, blob.index(), &blob.data[..])?;
         }
 
-        Ok(())
+        Ok(erasures.len())
```

Comment:
This should be `erasures.len() - 1` because there's a `-1` at the end to mark the end of erasures